.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,100 @@
+name: Build and test
+
+on:
+  push:
+      branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Having mutable ubuntu version might stright us back one day, I'd keep it fixed

edit: we have fixed ubuntu version run in docker. It can stay as it is here